### PR TITLE
[SELS-PREP] Setup local migration database

### PR DESCRIPTION
**Pull request by balagtasangelo**
[Commands]
            None
         [Pre-conditions]
            None
        [Expected Output]
             None
        [Notes]
                This is only a setup for the database using migration.

### CODE REVIEW

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -14,13 +14,13 @@ class CreateUsersTable extends Migration
     public function up()
     {
         Schema::create('users', function (Blueprint $table) {
+            $table->integer('user_id');
```
Comment (jeremiahC) [inline database/migrations/2014_10_12_000000_create_users_table.php]: I think dili naman ni needed guro.

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 2]
```diff
@@ -14,13 +14,13 @@ class CreateUsersTable extends Migration
...
             $table->id();
-            $table->string('name');
+            $table->boolean('is_admin');
             $table->string('email')->unique();
-            $table->timestamp('email_verified_at')->nullable();
+            $table->string('name');
             $table->string('password');
-            $table->rememberToken();
-            $table->timestamps();
```
Comment (jeremiahC) [inline database/migrations/2014_10_12_000000_create_users_table.php:23]: ayaw lang ni iremove, importante na naa tay created_at and update_at column para ma track nato ang pag save sa record

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 1 above)
Comment (balagtasangelo) [reply database/migrations/2014_10_12_000000_create_users_table.php]: Commited the requested changes. Please recheck.

File: database/migrations/2014_10_12_000000_create_users_table.php (hunk 2 above)
Comment (balagtasangelo) [reply database/migrations/2014_10_12_000000_create_users_table.php:23]: Commited the requested changes. Please recheck.

**Review by jeremiahC — CHANGES_REQUESTED** [review]
Please apply `timestamps()` function sa tanan table, para magamit nato ang `sort()` function sa laravel. created_at and updated_at kay gamit siya for sorting the data.

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**